src/registry.js

```diff
@@ -140,7 +140,13 @@ exports.traceIs = function(traceType, category) {
             Loggers.log('Unrecognized trace type ' + traceType + '.');
         }
 
-        _module = exports.modules[basePlotAttributes.type.dflt];
+        _module = exports.modules[dfltTrace];
+        if(!_module) {
+            Loggers.log('The default trace type ' + dfltTrace + ' is not registered.');
+
+            // in case the category starts with no return true otherwise false
+            return category.indexOf('no') === 0 ? true : false;
```

Comment:
Is it important to return `true` here for the `no` categories? In what situation would this occur?